python/pyspark/sql/connect/streaming/query.py

```diff
@@ -0,0 +1,181 @@
+#
```

Comment:
Should add this file to https://github.com/apache/spark/blob/master/dev/sparktestsupport/modules.py#L740. otherwise, the tests won't actually run in CI.